firedrake/utility_meshes.py

```diff
@@ -523,8 +526,12 @@ def IcosahedralSphereMesh(radius, refinement_level=0, reorder=None):
 
     :kwarg refinement_level: optional number of refinements (0 is an
         icosahedron).
+    :kwarg degree: polynomial degree of coordinate space (defaults
+        to 1: flat triangles)
     :kwarg reorder: (optional), should the mesh be reordered?
     """
+    if degree < 1:
+        raise RuntimeError("Mesh coordinate degree must be at least 1")
```

Comment:
Ah, fine.  Before writing this, I'd grepped for other `Error`s that we raised in this file, and found
```
    if length < 0:
        raise RuntimeError("Requested mesh has negative length")
```
so I went with RuntimeError.  Will change.